app/src/main/res/layout/activity_registration.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:gravity="center">
+    tools:context=".RegistrationActivity">
```

Comment:
this is not fine.. please delete it

Reply:
I did not delete this comand .... I wrote it correctly inside the tag